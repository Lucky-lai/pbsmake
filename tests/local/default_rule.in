
default: default_dep
	echo default rule is run

default_dep:
	VAR="dependency is run"
	echo $VAR

other:
	echo this should never be seen unless called explicitly
