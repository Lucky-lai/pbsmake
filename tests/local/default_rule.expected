dependency is run
default rule is run
